content/2017-01-08_pyladies-advanced.md

```diff
@@ -0,0 +1,20 @@
+Title: Pyladies Advanced: Srazy pro programováníchtivé dívky 
+Date: 2017-01-08 10:49:00 
+Author: Lenka 'Zouví' Matějová 
+Twitter: Zoellenka
+ Gravatar: zoelles@gmail.com
+About: Zou je holka, co se učí Python. Má stránky [Zoelles](http://zoelles.tumblr.com/), leze po skalách, chodí po slackline a miluje akrojógu.
+
+Od července do půle října 2016 probíhaly různě po Praze v kavárnách, posléze, díky Oskarovi,  v coworkingové zasedačce [Usertech](https://usertechnologies.com/) na Smíchově. Což je super prostor! Děkujeme!
+   
+Celkem proběhlo 22 těchto setkání, které vždy svolala nejlepší organizátorka Veronička.
+Pyladies Advanced bylo maloformátové setkání, nejvíce nás přišlo asi 12, někdy dorazilo i více koučů než Pyladies, ale umožnilo nám to: 
+1) posunout se dále v tom, co děláme. Často jsme díky pomoci koučů vyřešily problém s chybami, na které bychom samy došly jen těžko. Bylo nám vysvětleno spoustu nových témat, jako např. regulární výrazy.  Na tohle na kurzech většinou nezbývá čas a správné programátorce se to hodí znát! 
+  
+Materiálů je na internetu sice spousta, ale člověk se snadno zamotá. 
+2) Některá setkání byla více networkingového rázu typu meetup než programming session. Ale to nám zase umožnilo zapojit se aktivněji do Python komunity v ČR. Mohly jsme plánovat další kurzy pro budoucí Pyladies, podívat se na Pycon, zapojit se do psaní atd. 
+Na dvou Pyladies Advanced se objevila i zahraniční Pyladie, což je strašně super, že se tato mezinárodní komunita sdružuje. 
+Osobně jsem se snažila skoro žádné setkání Pyladies Advanced nezmeškat, i když ne vždy se mi to povedlo, protože to byla super akce, kde bylo vidět, jak se člověk posunuje v programování dále a učí se stále spousty nových věcí. Chtěla bych tímto velmi poděkovat koučům, že si na tato setkání našli čas a dorazili. Jste všichni skvělí! 
+  
```

Comment:
Ten blok se seznamem potřebuje trochu přeformátovat - když koukám na [náhled na GitHubu](https://github.com/zoellenka/blog.python.cz/blob/241afd287d088705b0406d12d0658ce320d47b31/content/2017-01-08_pyladies-advanced.md), Markdown si s tím neporadil. Tohle je obtížné i pro ostřílené Markdownové kovboje, tak s tím zkusím pomoct:

```
Celkem proběhlo 22 těchto setkání, které vždy svolala nejlepší organizátorka Veronička. Pyladies Advanced bylo maloformátové setkání, nejvíce nás přišlo asi 12, někdy dorazilo i více koučů než Pyladies, ale umožnilo nám to:

1.  posunout se dále v tom, co děláme. Často jsme díky pomoci koučů vyřešily problém s chybami, na které bychom samy došly jen těžko. Bylo nám vysvětleno spoustu nových témat, jako např. regulární výrazy.  Na tohle na kurzech většinou nezbývá čas a správné programátorce se to hodí znát!

    Materiálů je na internetu sice spousta, ale člověk se snadno zamotá. 

2.  Některá setkání byla více networkingového rázu typu meetup než programming session. Ale to nám zase umožnilo zapojit se aktivněji do Python komunity v ČR. Mohly jsme plánovat další kurzy pro budoucí Pyladies, podívat se na Pycon, zapojit se do psaní atd. 

Na dvou Pyladies Advanced se objevila i zahraniční Pyladie, což je strašně super, že se tato mezinárodní komunita sdružuje.
```

Výsledek:

------

Celkem proběhlo 22 těchto setkání, které vždy svolala nejlepší organizátorka Veronička. Pyladies Advanced bylo maloformátové setkání, nejvíce nás přišlo asi 12, někdy dorazilo i více koučů než Pyladies, ale umožnilo nám to:

1.  posunout se dále v tom, co děláme. Často jsme díky pomoci koučů vyřešily problém s chybami, na které bychom samy došly jen těžko. Bylo nám vysvětleno spoustu nových témat, jako např. regulární výrazy.  Na tohle na kurzech většinou nezbývá čas a správné programátorce se to hodí znát!

    Materiálů je na internetu sice spousta, ale člověk se snadno zamotá. 

2.  Některá setkání byla více networkingového rázu typu meetup než programming session. Ale to nám zase umožnilo zapojit se aktivněji do Python komunity v ČR. Mohly jsme plánovat další kurzy pro budoucí Pyladies, podívat se na Pycon, zapojit se do psaní atd. 

Na dvou Pyladies Advanced se objevila i zahraniční Pyladie, což je strašně super, že se tato mezinárodní komunita sdružuje.